examples/cstubs_structs/myocamlbuild.ml

```diff
@@ -0,0 +1,35 @@
+open Ocamlbuild_plugin
+
+let () =
+  let additional_rules = function
+    | Before_hygiene  -> ()
+    | After_hygiene   -> ()
+    | Before_options  -> ()
+    | After_options   -> ()
+    | Before_rules    -> ()
+    | After_rules     ->
+
+      (* Generate stubs. Steps 1, 2, & 3 of Makefile (1 & 2 via built-in rules).
+        ML -> C *)
+      rule "cstubs: x_c_gen.native -> x_stubs_gen.c"
+        ~dep:"%_c_gen.native"
+        ~prod:"%_stubs_gen.c"
+        (fun env _build -> Cmd (A (env "./%_c_gen.native")));
+                 (* OCAMLBULD_BUG: the './' is necessary *)
```

Comment:
Ugh, sorry that was meant more of a reminder for me to look into how (or if) `OCamlbuild` is setting `PATH`. I had worked on this over 2 periods and forgot about this in between the context switches. This isn't really a bug any more than the current directory isn't automatically included in the directories to search for executables. I'll remove the comment.